packages/blunderbuss/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "blunderbuss",
+  "version": "1.0.0",
+  "description": "randomly assigns issues or prs for triage",
+  "private": true,
+  "author": "Google Inc.",
+  "license": "Apache-2.0",
+  "repository": "https://github.com/googleapis/repo-automation-bots.git",
+  "homepage": "https://github.com/googleapis/repo-automation-bots",
+  "bugs": "https://github.com/googleapis/repo-automation-bots/issues",
+  "main": "build/src/app.js",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-app",
+    "lint"
+  ],
+  "scripts": {
+    "compile": "tsc -p .",
+    "start": "probot run ./build/src/blunderbuss.js",
+    "start:local": "node ./build/src/local.js",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "test": "cross-env LOG_LEVEL=fatal mocha --timeout=15000 --recursive ./build/test",
+    "test:snap": "SNAPSHOT_UPDATE=1 npm test",
+    "fix": "gts fix",
+    "lint": "gts check"
+  },
+  "dependencies": {
+    "@octokit/rest": "^16.28.3",
+    "gcf-utils": "^0.6.0",
+    "probot": "^9.3.0",
+    "js-yaml": "^3.12.1"
+  },
+  "devDependencies": {
+    "@types/body-parser": "^1.17.0",
```

Comment:
I'm a little surprised by all of the types packages here.  Are we using all of these?

Reply:
I bunch of these were required to get `probot` itself running, I think we can probably drop `@types/body-parser`, but the other types were required by probot.

```bash
node_modules/@octokit/app/dist-types/types.d.ts:1:22 - error TS7016: Could not find a declaration file for module 'lru-cache'. '/usr/local/google/home/bencoe/google/repo-automation-bots/packages/conventional-commit-lint/node_modules/lru-cache/index.js' implicitly has an 'any' type.
  Try `npm install @types/lru-cache` if it exists or add a new declaration (.d.ts) file containing `declare module 'lru-cache';`

1 import LRUCache from "lru-cache";
                       ~~~~~~~~~~~

node_modules/gcf-utils/build/src/gcf-utils.d.ts:17:26 - error TS7016: Could not find a declaration file for module 'express'. '/usr/local/google/home/bencoe/google/repo-automation-bots/packages/conventional-commit-lint/node_modules/express/index.js' implicitly has an 'any' type.
  Try `npm install @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`

17 import * as express from 'express';
                            ~~~~~~~~~

node_modules/probot/lib/application.d.ts:4:21 - error TS7016: Could not find a declaration file for module 'express'. '/usr/local/google/home/bencoe/google/repo-automation-bots/packages/conventional-commit-lint/node_modules/express/index.js' implicitly has an 'any' type.
  Try `npm install @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`

4 import express from 'express';
                      ~~~~~~~~~

node_modules/probot/lib/github/index.d.ts:1:23 - error TS2688: Cannot find type definition file for 'bunyan'.

1 /// <reference types="bunyan" />
                        ~~~~~~

node_modules/probot/lib/github/logging.d.ts:1:20 - error TS7016: Could not find a declaration file for module 'bunyan'. '/usr/local/google/home/bencoe/google/repo-automation-bots/packages/conventional-commit-lint/node_modules/bunyan/lib/bunyan.js' implicitly has an 'any' type.
  Try `npm install @types/bunyan` if it exists or add a new declaration (.d.ts) file containing `declare module 'bunyan';`

1 import Logger from 'bunyan';
```

seems like bad behavior to hoist the responsibility of installing our dependencies types on us, but it seems to be how things are structured right now.

I think we can potentially drop `@types/body-parser`.